kalite/main/tests/api_tests.py

```diff
@@ -1,10 +1,50 @@
 """
 """
+import json
+from django.core.urlresolvers import reverse
+
 from ..models import VideoLog, ExerciseLog
 from kalite.facility.models import Facility, FacilityUser
-from kalite.testing.base import KALiteTestCase
+from kalite.testing.base import KALiteTestCase, KALiteClientTestCase
 from kalite.testing.client import KALiteClient
 
+
+class ContentItemApiViewTestCase(KALiteClientTestCase):
```

Comment:
Okay, I'll do that. I don't understand in general how to know *what* content is available on the test server vs available to me locally. In this specific instance, I also have no idea what's supposed to be in the "extra fields", which was the cause of the original error.